home-excercises-05-2022/src/io/codelex/arithmetic/practice/SumAverageRunningInt.java

```diff
@@ -0,0 +1,19 @@
+package io.codelex.arithmetic.practice;
+
+import java.util.Locale;
+
+public class SumAverageRunningInt {
+    public static void main(String[] args) {
+        int sum = 0;
+        double average;
+        int lowerBound = 1;
+        int upperBound = 2;
+
+        for (int number = lowerBound; number <= upperBound; ++number) {
```

Comment:
For ciklos faktiski vienmēr raksta `number++` nevis `++number`, tehniski pamatojuma tur nav - tā vienkārši pieņemts.

Reply:
Sveiks! Šī daļa `++number` jau bija uzdota kā gatava - to nemainīju un pierakstīju klāt tikai lai izpildītos uzdevuma nosacījums ++ izteiksmi vispār nepamanīju, bet labāk izlabot, lai gan visdrīzāk tam ciklā nav atšķirības...

Reply:
Ok, to nepamanīju, ka tas jau bija uzdevumā. 👍

Reply:
🙏